acquia_purge_d8cache.drush.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * acquia_purge_d8cache.drush.inc
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function acquia_purge_d8cache_drush_command() {
+  $items = [];
+  // Clear all varnish cache command.
+  $items['apd8cache-flush'] = [
+    'description' => "Clear the Acquia Varnish cache for the site.",
+    'aliases' => ['apd8f'],
+    'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_FULL,
+  ];
+  return $items;
+}
+
+/**
+ * Drush callback to clear full site varnish caches.
+ */
+function drush_acquia_purge_d8cache_clear_site() {
```

Comment:
```suggestion
function drush_acquia_purge_d8cache_clear_site() {
```
function drush_acquia_purge_d8cache_apd8cache_flush() {

Reply:
thats now how suggestions work :man_facepalming:

Reply:
I fixd it

Reply:
🤷‍♂